packages/Form/summary/src/Summary.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import Alert from '@axa-fr/react-toolkit-alert';
+
+const defaultClassName = 'af-alert';
+const defaultClassModifier = 'danger';
+const defaultProps = {
+  title: 'Invalid form',
+  messages: null as string,
+  isVisible: false,
+  className: defaultClassName,
+  classModifier: defaultClassModifier,
+};
+
+interface MessagesProps {
+  messages: string[];
+}
+
+const Messages = ({ messages }: MessagesProps) => {
+  return (
+    <>
+      {messages.map(message => (
+        <li className="af-summary__message-item" key={message}>
+          <span>{message}</span>
+        </li>
+      ))}
+    </>
+  );
+};
+
+interface SummaryProps {
```

Comment:
I'am not typescript specialist but Types seem more adapted to the situation, you can check the explanation with @arnaudforaison who is the specialist :p

Reply:
https://medium.com/@martin_hotell/interface-vs-type-alias-in-typescript-2-7-2a8f1777af4c

Reply:
The tslint.json has the following: "interface-over-type-literal": true
Should I change it to "false"?